pkg/proxy/endpoints.go

```diff
@@ -200,6 +200,9 @@ func (ect *EndpointsChangeTracker) EndpointSliceUpdate(endpointSlice *discovery.
 			ect.lastChangeTriggerTimes[namespacedName] =
 				append(ect.lastChangeTriggerTimes[namespacedName], t)
 		}
+		klog.V(4).InfoS("EndpointSlice updated", "service", namespacedName, "endpointSlice", klog.KObj(endpointSlice))
+	} else {
+		klog.V(5).InfoS("EndpointSlice update does not affect proxying", "service", namespacedName, "endpointSlice", klog.KObj(endpointSlice))
```

Comment:
why not same log level, it should not be much difference of using 4 and 5, most people use level 2 by default and higher than 4 is should be ok to be verbose